.ci/Jenkinsfile

```diff
@@ -12,16 +12,8 @@ pipeline {
     BASE_DIR="src/github.com/elastic/elastic-package"
     JOB_GIT_CREDENTIALS = "f6c7695a-671e-4f4f-a331-acdce44ff9ba"
     PIPELINE_LOG_LEVEL='INFO'
-    AWS_ACCOUNT_SECRET = 'secret/observability-team/ci/elastic-observability-aws-account-auth'
     HOME = "${env.WORKSPACE}"
-    KIND_VERSION = 'v0.17.0'
-    K8S_VERSION = 'v1.26.0'
-    JOB_GCS_BUCKET = 'fleet-ci-temp'
-    JOB_GCS_BUCKET_INTERNAL = 'fleet-ci-temp-internal'
-    JOB_GCS_CREDENTIALS = 'fleet-ci-gcs-plugin'
-    JOB_GCS_EXT_CREDENTIALS = 'fleet-ci-gcs-plugin-file-credentials'
-    ELASTIC_PACKAGE_GCP_SECRET = 'secret/observability-team/ci/service-account/elastic-package-gcp'
-    ELASTIC_OBSERVABILITY_PROJECT_ID = 'elastic-observability'
+    BUILDKITE_MERGE_COMMIT = "79ee1738578deac17f8f5caf8d925db6244ab6df"
```

Comment:
Commit reference for the merge commit where the buildkite pipeline was merged:
https://github.com/elastic/elastic-package/commit/79ee1738578deac17f8f5caf8d925db6244ab6df